tests/browser/test/basic_playground_test.js

```diff
@@ -100,14 +103,14 @@ suite('Right Clicking on Blocks', function (done) {
 
   test('clicking the disable option disables the block', async function () {
     await contextMenuSelect(browser, this.block, 'Disable Block');
-    const isEnabled = await getIsEnabled(browser, this.blockId);
-    chai.assert.isFalse(isEnabled);
+    const isDisabled = await getIsDisabled(browser, this.blockId);
+    chai.assert.isTrue(isDisabled);
```

Comment:
Any reason not to combine these into a single statement?
```suggestion
    chai.assert.isTrue(await getIsDisabled(browser, this.blockId));
```

Reply:
I was trying to match the format that Rachel used.  But I'll go through and change all of 'em!

Reply:
fwiw, I used separate lines because I like having `await` at the beginning of the line.